tests/test_convertors.py

```diff
@@ -0,0 +1,99 @@
+import datetime as dt
```

Comment:
I've created multiple convertors here with the purpose of reproducing https://github.com/encode/starlette/pull/1398. I can also simplify the test suite if wanted. Just let me know.

Reply:
I agree with simplifying the tests, as this can be extended for other things like `UUID` and there's not really much difference. This is really good.